tools/image-builder/build_image.zsh

```diff
@@ -3,4 +3,4 @@
 set -eux
 
 docker build -t bond-xenial .
-docker save bond-xenial | xz > bond-xenial.tar.xz
```

Comment:
Would be nice to `time` here, too. i.e., `| time xz`.

Reply:
Updated.